lobby/src/main/java/org/triplea/lobby/server/login/forgot/password/create/PasswordEmailSender.java

```diff
@@ -0,0 +1,57 @@
+package org.triplea.lobby.server.login.forgot.password.create;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Properties;
+import java.util.function.BiConsumer;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import org.triplea.lobby.server.EnvironmentVariable;
+
+/** Sends a temporary password to a target user. */
+class PasswordEmailSender implements BiConsumer<String, String> {
+  private static final String FROM = "no-reply@triplea-game.org";
+
+  @Override
+  public void accept(final String email, final String generatedPassword) {
+    if (EnvironmentVariable.LOCAL_DEV.getBoolean()) {
+      // do not send emails from local dev.
+      return;
+    }
+
+    final Properties props = new Properties();
+    final Session session = Session.getDefaultInstance(props, null);
+
+    try {
+      final Message message = new MimeMessage(session);
+      message.setFrom(new InternetAddress(FROM, "no-reply"));
+      message.addRecipient(Message.RecipientType.TO, new InternetAddress(email, email));
+      message.setSubject("Temporary Password");
+      message.setText(createMailBody(generatedPassword));
```

Comment:
Perhaps out of scope for this PR but plain text is really ugly.
This should really be a multipart email.
Have a look at some simple templates I created for the NodeJS replacement of MARTI: https://github.com/triplea-game/dice-server-js/tree/master/public

Reply:
The email should not be frequent, IMO a short plain text looked okay enough to me :smile: 

If we want to improve it, sure :man_shrugging: 
The testing would need to be repeated though; after banging my head against encryption for around 9 hours, and banging my head against the client/server quarantine conversation for about 5, I'm very ready to move on :smiley: